tools/images/ubuntu_alt/Dockerfile

```diff
@@ -4,20 +4,16 @@ MAINTAINER peter.ebden@gmail.com
 # Most dependencies; Python, Java, Clang and Ruby (only used for fpm)
 RUN apt-get update && \
     apt-get install -y python3 python3-dev python3-pip openjdk-11-jdk-headless \
-    curl unzip git locales pkg-config zlib1g-dev \
-    clang libunittest++-dev ruby ruby-dev rubygems golint && \
+    curl unzip git locales pkg-config zlib1g-dev clang && \
```

Comment:
This is now missing golint - maybe keep these lists in alphabetical order?

Reply:
That was deliberate - I thought I didn't need it. Have updated CircleCI so I really don't (there's no point linting in two different steps)